docs/docs-source/docs/modules/ROOT/nav.adoc

```diff
@@ -12,7 +12,7 @@ include::partial$include.adoc[]
 *** xref:get-started:develop-example-streamlets.adoc[Develop example streamlets]
 *** xref:get-started:create-example-blueprint.adoc[Create the example blueprint]
 ** xref:get-started:run-in-sandbox.adoc[Run in a local sandbox]
-** xref:get-started:setup-gke-cluster.adoc[Set up a GKE cluster]
+** xref:get-started:setup-k8s-cluster.adoc[Set up a Kubernetes cluster]
```

Comment:
Why is this changed from `GKE` to `Kubernetes`? The rest of the text refers to GKE.

Reply:
Because the instructions in the original `setup-gke-cluster.adoc` file are mostly generic that applies to other K8s flavors too, except for a few lines that are GKE specific. And the installer is also flavor-agnostic.